### DISCUSSION TITLE
Is there any consideration of adding tmux control mode support to Windows Terminal like iTerm2 has?

### DISCUSSION THREAD

**Discussion by fourpastmidnight** [microsoft/terminal#15613] [Ideas]
In short, `tmux -CC` (`tmux` control mode) allows you to use your terminal emulator to send keys to `tmux` to control it. Many terminals either have support (like iTerm2) or have open items to add such support. See [this Unix & Linux SO Q&A](https://unix.stackexchange.com/questions/189805/what-terminal-emulators-support-tmux-control-mode).

(Perhaps I should have opened this up as a poll instead...)

**Comment by j4james** [comment]
Yes, there is already an open issue for tmux control mode. See #3656.

**Reply by fourpastmidnight** [reply]
Ah, thanks. I didn't think to check the issues!

**Comment by fin2b3pol** [comment]
Thanks for the work
